### PR TITLE
remove the need to pass in browser as parameter to the methods

### PR DESCRIPTION
**Pull request by titusfortner**
Rewrite of #31 now that Watir 6.12 has (finally) been released.

@samnissen your blogpost for the website got auto-deleted when we force-pushed the new way of doing the website build. When you get the new release of this gem out, can you push up a new version rebased with source branch, and removing the need to pass in browser in the examples?

Thanks!

### CODE REVIEW

File: lib/watir-screenshot-stitch.rb [hunk 1]
```diff
@@ -78,7 +73,7 @@ def one_shot?
       end
 
       def bug_shot?
-        return false unless @browser&.name == :firefox
```
Comment (samnissen) [inline lib/watir-screenshot-stitch.rb]: I'm curious what the motivation for removing the safety is here?

File: README.md [hunk 2]
```diff
@@ -98,7 +90,7 @@ TODO: This.
 
 ## Contributing
 
-Bug reports and pull requests are welcome on GitHub at https://github.com/[USERNAME]/watir-screenshot-stitch. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
+Bug reports and pull requests are welcome on GitHub at https://github.com/samnissen/watir-screenshot-stitch. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
```
Comment (samnissen) [inline README.md]: Good catch -- thank you

File: lib/watir-screenshot-stitch.rb (hunk 1 above)
Comment (titusfortner) [reply lib/watir-screenshot-stitch.rb]: I just pushed a commit to change this back and set minimum Ruby to 2.3. Watir hasn't enforced that, yet, but we'll be doing it soon.

**Comment by titusfortner** [comment]
Sounds good. When you get it released, make another PR on our website code with your blog post, and we'll get that published/promoted.

**Comment by sandeepnagra** [comment]
@titusfortner - could you please resolve the conflicts so that this PR can be merged?

**Comment by sandeepnagra** [comment]
> @titusfortner - could you please resolve the conflicts so that this PR can be merged?
@samnissen - just noticed that #47 is taking care of removing browser as parameter, do you you want to close this PR?

**Comment by samnissen** [comment]
@titusfortner @sandeepnagra I believe we should close this once #47 is merged.

**Comment by samnissen** [comment]
Closing per comments above

**Pull request closed without merging**